src/pages/pattern-library.tsx

```diff
@@ -141,7 +141,7 @@ function Page() {
                 <UIBadge>15% off</UIBadge>
               </li>
               <li>
-                <UIBadge className="badge_small">15% off</UIBadge>
+                <UIBadge data-store-badge="small">15% off</UIBadge>
```

Comment:
Shouldn't we use the `DiscountBadge` component here? something like:

```js
<li>
  <DiscountBadge listPrice={10} spotPrice={5} />
</li>
<li>
  <DiscountBadge listPrice={10} spotPrice={5} small />
</li>
```

* with the imports.

Reply:
You are right! I think it makes more sense! I'll update it! :)